Configuration/src/ConfigurationProviders/config/application-configuration-service/3-claim.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿apiVersion: "services.apps.tanzu.vmware.com/v1alpha1"
 kind: ResourceClaim
 metadata:
- name: my-config
+ name: sample-config-server
```

Comment:
This ResourceClaim is for Application Configuration Service, not config server, so this name could be confusing